gulpfile.js

```diff
@@ -79,9 +84,25 @@ function bowerTask() {
 
 function buildTask() {
 
+  var errorHandler = function (err) {
+    util.log(util.colors.red('[Error]'), err.toString());
```

Comment:
I would move that line before `this.emit('end');`, it's only useful when ignoring exceptions